main.go

```diff
@@ -76,25 +76,40 @@ func cacheYarn(workingDir string) error {
 	}
 
 	if err := yarnCache.Commit(); err != nil {
-		return fmt.Errorf("failed to mark node_modeules directories to be cached, error: %s", err)
+		return fmt.Errorf("failed to mark node_modules directories to be cached: %s", err)
 	}
 	return nil
 }
 
 func main() {
 	var config config
 	if err := stepconf.Parse(&config); err != nil {
-		failf("Invalid input: %s", err)
+		failf("Process config: %s", err)
 	}
 	stepconf.Print(config)
 	fmt.Println()
 	log.SetEnableDebugLog(config.IsDebugLog)
 
+	absWorkingDir, err := filepath.Abs(config.WorkingDir)
+	if err != nil {
+		failf("Process config: faield to normalize working directory: %s", err)
+	}
+
+	commandParams, err := shellquote.Split(config.YarnCommand)
+	if err != nil {
+		failf("Process config: invalid quoting of provided yarn command: %s", err)
```

Comment:
How about this? "Invalid quoting" is hard to understand without seeing the actual source code.

```suggestion
		failf("Process config: provided yarn command is not a valid CLI command: %s", err)
```